proposals/interceptors-mvp.md

```diff
@@ -0,0 +1,32 @@
+Interceptors MVP:
+
+- generated SyntaxTree.FilePath
```

Comment:
```suggestion
- generated `SyntaxTree.FilePath`
```

Is this referring to the file path for generated `SyntaxTree` being the "what we would write" generated path.